src/main/java/ti4/generator/TileGenerator.java

```diff
@@ -74,7 +73,7 @@ public TileGenerator(Game game, GenericInteractionCreateEvent event) {
     }
 
     public TileGenerator(Game game, @Nullable GenericInteractionCreateEvent event, @Nullable DisplayType displayType,
-                         int context, String focusTile) {
```

Comment:
We should keep these spaced over to avoid them being on the same line as the code within the method.

Reply:
i think this was just automatically done by the vscode lint settings